plugins/SoundFontPlayer/SoundFontPlayer.cpp

```diff
@@ -910,11 +910,11 @@ void Sf2Instrument::renderFrames( f_cnt_t frames, SampleFrame* buf )
 #endif
 		if( error )
 		{
-			qCritical( "Sf2Instrument: error while resampling: %s", src_strerror( error ) );
+			qCritical( "SoundFontInstrument: error while resampling: %s", src_strerror( error ) );
```

Comment:
(Non-blocking) Per the Coding Conventions [re: parentheses](https://github.com/LMMS/lmms/wiki/Coding-Conventions#braces):

> 1. Spaces MUST NOT be added after an opening parenthesis or bracket
> 2. Spaces MUST NOT be added before a closing parenthesis or bracket

```suggestion
			qCritical("SoundFontInstrument: error while resampling: %s", src_strerror(error));
```